lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -272,6 +272,47 @@ bool isTppZero(linalg::GenericOp linalgOp, SmallVectorImpl<Value> *operands) {
   return true;
 }
 
+SmallVector<Operation *> splitFusedOp(tpp::FusedBrgemmOp fusedBrgemmOp,
+                                      PatternRewriter &rewriter) {
+  assert(fusedBrgemmOp.hasBufferSemantics() &&
+         "tpp.fused_brgemm expects a memref type");
+
+  SmallVector<Operation *> splitOps;
+  Location loc = fusedBrgemmOp.getLoc();
+
+  // Split the fused op into individual operations.
+  auto ins = fusedBrgemmOp.getInputs();
+  auto out = fusedBrgemmOp.getOutput();
+  Operation *brgemm = rewriter.create<tpp::BrgemmOp>(
```

Comment:
This function creates new operations but requires the caller to remove the old one, I don't see a case where this would be beneficial over erasing it here, and it's not needed in the one case where you use it.